web/src/beta/lib/core/Visualizer/index.tsx

```diff
@@ -211,6 +218,8 @@ export default function Visualizer({
           overriddenClock={overriddenClock}
           blocks={infobox?.blocks}
           camera={camera}
+          interactionMode={interactionMode}
+          overrideInteractionMode={handleInteractionModeChange}
```

Comment:
Is it possible to get core to adhere to our style guide?
I am not sure why we decided to (since core's development happened after the style guide was in place), but core often ignores even common React standards like callback declarations being named `handleXxxxx` and callback props being named `onXxxxxx`.
Because of this, reading the code without hover for type descriptions is hard to know what is a variable and what is a function..

Reply:
P.S I know this is out of the scope of this PR. Just wanted to ask

Reply:
The reason is these functions are used as Plugin API as well. If you prefer to use `handleXxxx`, I will fix it!